components.json

```diff
@@ -1196,6 +1196,12 @@
 			"description": "Highlights the code inside diff blocks.",
 			"owner": "RunDevelopment",
 			"require": "diff"
+		},
+		"not-all": {
```

Comment:
Some thoughts on the name:

* `filter-elements`
* `modify-highlighted`
* `filter-highlighted`

Thoughts?

Reply:
I want to have _all_ in the name, so people associate it with `highlightAll`.

- `filter-all-elements`
- `filter-highlight-all`

(Also, no past tense (`filter-highlighted`). They're not highlighted yet ;) )

Reply:
`filter-highlight-all` works for me!